### ISSUE TITLE
TypeError: Cannot redefine property: usingDomains

### ISSUE THREAD

**Issue by kqkq** [hexojs/hexo#1165]
Hexo is completely failed after update to 3.0.

`hexo g`/`hexo s`/`hexo clean` will generate:

```
ERROR TypeError: Cannot redefine property: usingDomains
    at Function.defineProperty (native)
    at Object.<anonymous> (/Users/xxxxx/Desktop/hexo3/node_modules/hexo/node_modules/bluebird/js/main/async.js:75:16)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at module.exports (/Users/xxxxx/Desktop/hexo3/node_modules/hexo/node_modules/bluebird/js/main/promise.js:13:13)
    at Object.<anonymous> (/Users/xxxxx/Desktop/hexo3/node_modules/hexo/node_modules/bluebird/js/main/bluebird.js:9:39)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
Unhandled rejection TypeError: Cannot read property 'then' of undefined
    at /usr/local/lib/node_modules/hexo-cli/lib/index.js:82:21
    at tryCatcher (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/util.js:24:31)
    at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/promise.js:454:31)
    at Promise._settlePromiseAt (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/promise.js:530:18)
    at Async._drainQueue (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/async.js:169:12)
    at Async._drainQueues (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/async.js:174:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/lib/node_modules/hexo-cli/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```

**Comment by naokiy** [comment]
I think this is a problem of bluebird.
see https://github.com/petkaantonov/bluebird/issues/559 .

**Comment by tommy351** [comment]
It seems bluebird fixed it in 2.9.24. Try to install Hexo again.

**Comment by kqkq** [comment]
Solved
